packages/website/ts/components/eth_wrappers.tsx

```diff
@@ -345,9 +383,14 @@ export class EthWrappers extends React.Component<EthWrappersProps, EthWrappersSt
         });
     }
     private getOutdatedWETHAddresses(): string[] {
-        const outdatedWETHAddresses = _.map(configs.outdatedWrappedEthers, outdatedWrappedEther => {
-            return outdatedWrappedEther[this.props.networkId].address;
-        });
+        const outdatedWETHAddresses = _.compact(_.map(configs.outdatedWrappedEthers, outdatedWrappedEtherByNetwork => {
+            const  outdatedWrappedEtherIfExists = outdatedWrappedEtherByNetwork[this.props.networkId];
```

Comment:
there's an extra space after const here

Reply:
Huh, weird the linter isn't catching that. Any ideas @LogvinovLeon?